arrows/cuda/integrate_depth_maps.cxx

```diff
@@ -186,11 +186,11 @@ copy_img_to_gpu(kwiver::vital::image_container_sptr h_img)
 
   //copy to cuda format
   kwiver::vital::image img = h_img->get_image();
-  for (unsigned int i = 0; i < h_img->width(); i++)
+  for (unsigned int j = 0; j < h_img->height(); j++)
```

Comment:
This is a very weak suggestion but I found the use of `i` and `j` relatively confusing because they don't agree with the matrix meaning. Perhaps `x` and `y` as in the next section of edited code would be better.

Reply:
This notation comes from VXL where it is used heavily.  I'm trying to be consistent with that.  The accessors for width and height on a VXL image are `ni()` and `nj()`.  This is the old row-major vs column major debate.